sageresearch-app-sdk/src/androidTest/java/org/sagebionetworks/research/sageresearch/viewmodel/ReportRepositoryTests.kt

```diff
@@ -91,6 +90,10 @@ class ReportRepositoryTests: RoomTestHelper() {
         val bridgeGson = EntityTypeConverters().bridgeGson
     }
 
+    @Rule
+    @JvmField
+    var testSchedulerRule = TestSchedulerRule()
```

Comment:
I don't understand how this fixes anything because you don't use this variable anywhere.